lib/entities/release.ts

```diff
@@ -80,12 +81,24 @@ export type ReleaseSysProps = {
   updatedAt: ISO8601Timestamp
   lastAction?: Link<'ReleaseAction'>
 }
-
+export type ReleasesReferenceFilters = ScheduledActionReferenceFilters
+
+export type MetadataAggeratedRootProps = {
+  metadata: {
+    withReferences: [
+      {
+        entity: Link<'Entry'>
+        filter: Record<ScheduledActionReferenceFilters, string[]>[]
+      }
+    ]
+  }
+}
```

Comment:
```suggestion
export type MetadataAggeratedRootProps = {
    withReferences: [
      {
        entity: Link<'Entry'>
        filter: Record<ScheduledActionReferenceFilters, string[]>[]
      }
    ]
}
```

delete `metadata` object as its already there in `ReleaseProps`

Reply:
okay will do thanks!